platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -128,9 +128,13 @@
     THERMAL_API_GET_HIGH_THRESHOLD
 ]
 
-platform_dict_thermal = {'x86_64-mlnx_msn2700-r0': 0, 'x86_64-mlnx_lssn2700-r0':0, 'x86_64-mlnx_msn2740-r0': 3, 'x86_64-mlnx_msn2100-r0': 1, 'x86_64-mlnx_msn2410-r0': 2, 'x86_64-mlnx_msn2010-r0': 4, 'x86_64-mlnx_msn3420-r0':9, 'x86_64-mlnx_msn3700-r0': 5, 'x86_64-mlnx_msn3700c-r0': 6, 'x86_64-mlnx_msn3800-r0': 7, 'x86_64-mlnx_msn4600c-r0':9, 'x86_64-mlnx_msn4700-r0': 8, 'x86_64-mlnx_msn4410-r0': 8}
+platform_dict_thermal = {'x86_64-mlnx_msn2700-r0': 0, 'x86_64-mlnx_lssn2700-r0': 0, 'x86_64-mlnx_msn2740-r0': 3,
+                         'x86_64-mlnx_msn2100-r0': 1, 'x86_64-mlnx_msn2410-r0': 2, 'x86_64-mlnx_msn2010-r0': 4,
+                         'x86_64-mlnx_msn3420-r0': 9, 'x86_64-mlnx_msn3700-r0': 5, 'x86_64-mlnx_msn3700c-r0': 6,
+                         'x86_64-mlnx_msn3800-r0': 7, 'x86_64-mlnx_msn4600-r0': 12, 'x86_64-mlnx_msn4600c-r0': 9,
+                         'x86_64-mlnx_msn4700-r0': 8, 'x86_64-mlnx_msn4410-r0': 8}
 thermal_profile_list = [
-    # 2700
+    # 0 2700
```

Comment:
what is do you need a leading number in the comment?

Reply:
The platform_dict_thermal is supposed to have the index of the switch thermal_profile_list, I added those to help me find the index of my switch in the array.

I kept it because it having this information in front of our eyes helped me realize there is a mistake in other switches (x86_64-mlnx_msn3420-r0 having 9 as its index when in fact it is in the 11th place of the array, 4410 which is in the 11 place of the list is missing from this array at all).

I can remove it if it is not needed